apps/service_define/lib/define/view_state.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Define.ViewState do
```

Comment:
I'm not clear what this module is doing but the term "ViewState" is pretty loaded in our project to mean application state persisted to Redis or Postgres via the Brook library. Is this a stub for that that will be filled out in later work?